src/shared/Elements/ConfirmModal.tsx

```diff
@@ -38,7 +40,7 @@ export const ConfirmModal: React.FC<IConfirmModalProps> = (
               onClick={props.onConfirmed}
               variant={ButtonVariant.Primary}
             >
-              Yes
+              {props.confirmLabel || "Yes"}
```

Comment:
Can "yes" and "no" be default values of the props?

Reply:
oops yeah i'll change it asap!